### ISSUE TITLE
add export/download logs from Logs explorer screen

### ISSUE THREAD

**Issue by thb-bahaa** [SigNoz/signoz#3331]
## Bug description

Logs explorer screen didn't have option to download the logs

## Expected behavior

Logs explorer screen should have option to download the logs.

## How to reproduce

1.deploy signoz chart version 0.21.0

## Version information 
* **Signoz version**: 0.25.0
* **Browser version**: Microsoft edge
* **Your OS and version**: the sinoz installed on kubernates using helm
* **Your CPU Architecture**(ARM/Intel):  N/A

**Comment by pranay01** [comment]
Hey @thb-bahaa 

In the main logs explorer page, you should be able to see the `Download` options (as shown in screenshot below) or are you looking for a download option in the new logs explorer page?

![logs-explorer](https://github.com/SigNoz/signoz/assets/504541/e714a651-35af-42f2-bbe6-1b7989d4f51f)

**Comment by thb-bahaa** [comment]
@pranay01 yes brother you are correct, i mean from new logs explorer page

**Comment by palashgdev** [comment]
ideally, the download logs feature should prompt the user in which timeframe the user wanna download the logs and the backend would send the CSV file

**Comment by palashgdev** [comment]
@nityanandagohain @ankitnayan ?

**Comment by palashgdev** [comment]
https://github.com/SigNoz/signoz/pull/3848#issuecomment-1787486081

**Comment by nityanandagohain** [comment]
Yeah making this backend driven will make more sense since as of now the current feature just allows the user to download the rendered data.

**Comment by palashgdev** [comment]
would recommended >  user wanting to download data in a timeframe with filters.

**Comment by pranay01** [comment]
@GeekBoySupreme For the current download option you have designed, what is it's expected behavior? Just download current rendered data or ask them for a timeframe to filter from?

cc @nityanandagohain

**Comment by nityanandagohain** [comment]
Duplicate of https://github.com/SigNoz/signoz/issues/3331. We will prioritise this.
